doc/doxygen/api/doxygen.config

```diff
@@ -1542,7 +1542,7 @@ MATHJAX_CODEFILE       =
 # The default value is: YES.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-SEARCHENGINE           = NO
+SEARCHENGINE           = YES
```

Comment:
SEARCHENGINE is turned on here for API dox, but turned off for BSP dox, is this discrepancy intentional?

Reply:
Nope - good catch. Fixed :)